modulo-5/ejercicios/56-fundamento-sass.md

```diff
@@ -0,0 +1,248 @@
+5. Cambiamos la variable por un valor `true`
+6. Volvemos a validar los cambios.
+
+### Ejercicio 2 - Operadores de comparación
+
+1. En el archivo de sass crear las variables `$height:120px` y `$width:150px`
+2. En el html creamos una caja
+3. A la caja le asignamos las variables que creamos para el alto y para el ancho
+4. Usamos un condicional y preguntamos si el alto es más grande que el ancho, y lo resolvemos de la siguiente forma:
+   a. Si es verdadero el color de fondo sea un azul
+   b. Si es falso que sea de color rojo
+5. Cambiar el valor de las variables para que se vea de color azul la caja.
+
+### Ejercicio 3 - Operadores de comparación y lógicos
+
+1. Repetimos el ejercicio 2
+2. Ahora el condicional tiene que ser si el alto es igual o más chico que el ancho y además el ancho no supere los 121 px se deberá resolver de la siguiente forma:
+   a. Si es verdadero el color de fondo sea un azul
+   b. Si es falso que sea de color rojo
+3. Cambiar el valor de las variables para que se vea de color azul la caja.
+
+### Ejercicio 4 - Operadores de comparación y lógicos
+
+1. Repetimos el ejercicio 2
+2. Ahora el condicional tiene que ser si el alto es diferente al ancho y además el ancho sea igual a 120 px se deberá resolver de la siguiente forma:
+   a. Si es verdadero el color de fondo sea un azul
+   b. Si es falso que sea de color rojo
+3. Cambiar el valor de las variables para que se vea de color azul la caja.
+
+## Anidado
+
+### Base
+
+Crean un archivo html y usando emmet replicar el siguiente código
+`(div>(p>span>lorem)+(p*3>lorem)+(p>span>lorem))+(p*2>span>lorem+span>lorem)+(h1>lorem10)+p*2>span>lorem+span>lorem`
+Para esto hay que vscode, hay que copiarlo y pegarlo.Luego apretar la combinación de teclas `ctrl+ barra espaciadora` o `⌃ + barra espaciadora` en mac.Esto despliega la sugerencia apretar enter y se le creara un código HTML necesario para los ejercicios.
+
+### Parte 1
+
+Usando combinadores de espacio en Sass darle los siguientes estilos:
+
+- A las `p` que están dentro del div dar un color de fondo `blue`
+- A los `span` que están dentro del div darle un color de fondo `red`
+
+### Parte 2
+
+Usando combinadores de hijo(>) en Sass darle los siguientes estilos:
+
+- A los `span` que son hijos directos del una etiqueta `p` cambiar el color a un `green`.
+
+### Parte 3
+
+Usando combinadores de hermanos adyacentes(+) en Sass darle los siguientes estilos:
+
+- A los `p` que son hermanos adyacentes del `h1` cambiar el tamaño de letra por `24px`
+
+### Parte 4
+
+Usando combinadores generales de hermanos(~) en Sass darle los siguientes estilos:
+
+- A los `p` que son hermanos generales del `div` cambiar el fondo de color por `silver`
+
+### Selector padre
+
+### Base
+
+Crear los archivos necesarios y utilizar la siguiente plantilla de emmet:
+`.card>(.card__header>h3{Soy un card})+(.card__body>p>lorem35)+(.card__footer>button.button.button--primary{Aceptar}+button.button.button--danger{Cancelar})`
```

Comment:
Lo mismo que el [comentario anterior](https://github.com/Ada-IT/ejercicios-frontend/pull/3/commits/512416b7297522b64e2b9a94021ade2ded4c995a#r500292221)